e2e_tests/tests/cluster/test_checkpoints.py

```diff
@@ -47,7 +47,6 @@ def wait_for_gc_to_finish(sess: api.Session, experiment_ids: List[int]) -> None:
 
 
 @pytest.mark.e2e_gpu
```

Comment:
does this need to be gpu even?

Reply:
same for `test_gc_checkpoints` here

Reply:
same for `test_s3_no_creds` (though it's being skipped it seems)

Reply:
that might be a question for @stoksc, I think maybe that's to run those tests on fewer clusters?

Reply:
wouldn't `e2e_cpu` cause it to run on fewer clusters?